SharePointFramework/PortfolioWebParts/src/components/PortfolioOverview/index.tsx

```diff
@@ -446,7 +446,7 @@ export class PortfolioOverview extends Component<IPortfolioOverviewProps, IPortf
         throw new PortfolioOverviewErrorMessage(strings.ViewNotFoundMessage, MessageBarType.error)
       }
     } else if (defaultViewId) {
-      currentView = _.find(views, (v) => v.id.toString() === defaultViewId)
+      currentView = _.find(views, (v) => v.id.toString() === defaultViewId.toString())
```

Comment:
Isn't `defaultViewId` a string already?

Reply:
cc @Petterandre

Reply:
It should be, but i logged typeof(defaultViewId) and for some reason it was number. VSCode says it is defined as a string, but at runtime it isn't for some reason.

Reply:
![image](https://user-images.githubusercontent.com/10527497/114534937-ce27db00-9c4f-11eb-9956-80f70ad907fe.png)